example/golang.mk

```diff
@@ -1,7 +1,7 @@
 # Source: https://github.com/rebuy-de/golang-template
-# Version: 1.3.2-snapshot
+# Version: 1.4.1-snapshot
 # Dependencies:
-# * Glide
+# * Godep
```

Comment:
`godep != dep`

Reply:
fixed it https://github.com/rebuy-de/golang-template/pull/12